_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
-msgstr ""
+msgstr "**Windows-Benutzer (ASIO4ALL)**: Wenn du den ASIO4ALL-Treiber verwendest, wirf einen Blick auf den [ASIO4LL einrichten](Installation-for-Windows#asio4all-einrichten)\n"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:27
 msgid "### Not seeing the headphone or microphone you just connected to your PC?"
-msgstr ""
+msgstr "### Du kannst den Kopfhörer oder das Mikrofon nicht sehen, den/das du gerade an deinen PC angeschlossen hast?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:29
 msgid "Just restart Jamulus with your device plugged in. Jamulus currently doesn't show devices which were plugged in after the sound system was loaded."
-msgstr ""
+msgstr "Starte Jamulus einfach neu, wenn deine Geräte eingesteckt sind. Jamulus zeigt derzeit keine Geräte an, die nach dem Laden des Soundsystems eingesteckt wurden."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:31
 msgid "### Your sound is stuttering"
-msgstr ""
+msgstr "### Dein Sound stottert"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:33
 msgid "Your audio device may not work with the buffer size you selected. Select a bigger buffer size in Jamulus's settings. If possible though, use another device, since large buffer sizes mean higher latency."
-msgstr ""
+msgstr "Dein Audiogerät funktioniert möglicherweise nicht mit der von dir gewählten Puffergröße. Wähle in den Einstellungen von Jamulus eine größere Puffergröße. Wenn möglich, solltest du ein anderes Gerät verwenden, da große Puffergrößen eine höhere Latenz bedeuten."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:35
 msgid "### You all sound OK, but it's difficult to keep together"
-msgstr ""
+msgstr "### Der Sound ist gut aber es ist schwierig im Takt zu bleiben"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:37
 #, no-wrap
 msgid "**If at all possible, DO NOT listen to your direct signal.** Make sure you are listening as much as you can to the sound of your own instrument/voice _coming back from the Server_. This signal has you and your playing partners mixed together in sync, and will be the signal that your bandmates are hearing too. So listening to that means you will be in sync with each other (assuming you all have reasonably low latency). Note that if one or more musicians are not following this rule, they will slow down as they play or sing.\n"
-msgstr ""
+msgstr "**Wenn möglich, höre NICHT auf dein direktes Signal.** Stelle sicher, dass du so viel wie möglich auf den Klang deines eigenen Instruments/Stimme hörst, der _vom Server_ zurückkommt. Dieses Signal wird von dir und deinen Mitspielern synchron gemischt und ist das Signal, das auch deine Bandkollegen hören. Wenn du dieses Signal hörst, seid ihr untereinander synchron (vorausgesetzt, alle haben eine einigermaßen geringe Latenz). Beachte, dass ein oder mehrere Musiker, die sich nicht an diese Regel halten, beim Spielen oder Singen langsamer werden.\n"
```

Comment:
not so viel wie möglich but 
so stark wie möglich or so gut wie möglich

Reply:
Corrected